test/crypto-browserify.test.js

```diff
@@ -1,9 +1,10 @@
-const crypto = require('crypto');
```

Comment:
Is it possible to remove crypto from the dependencies too?

Reply:
`crypto` is a native node module